source/viz.md

```diff
@@ -1219,22 +1307,36 @@ Bar plot of size for Earth's largest 12 landmasses colored by whether its a cont
 :::
 
 
-The plot in {numref}`islands_plot_sorted` is now a very effective
+The plot in {numref}`islands_plot_sorted` is now an effective
 visualization for answering our original questions. Landmasses are organized by
 their size, and continents are colored differently than other landmasses,
-making it quite clear that continents are the largest seven landmasses.
-We can make one more finishing touch in {numref}`islands_plot_titled`: we will
-add a title to the chart by specifying `title` argument in the `alt.Chart` function.
-Note that plot titles are not always required; usually plots appear as part
+making it quite clear that all the seven largest landmasses are continents.
+
+To add some finishing touches to the chart
+we will add a title to the chart by specifying `title` argument inside `alt.Chart`.
+A good plot title should usually contain the take home message that you want readers of the chart to focus on,
+e.g. "The Earth's seven largest landmasses are all continents",
+but it could sometimes be more general, e.g. "The twelve largest landmasses on Earth".
```

Comment:
NB: this stuff about titles seems to have been removed from the R version. I put a pretty high value on syncing the two books as much as possible to avoid divergent material, so we should decide whether this is included or removed or de-synced from R

Reply:
(though obviously there will be differences due to differing syntax / code patterns etc, but the set of "ideas/topics we teach" should flow the same)

Reply:
I agree that it would be valuable to keep them in sync and would prefer to have this section added to the R book. I think it is important to mention titles and show examples of what an appropriate title looks like.

Reply:
@joelostblom it took me a bit to dig around in my memory for why we decided against this in R, but I finally remembered: we don't use titles anywhere else in the book. If we're suggesting Ch4 is the guide to making good viz, we should follow our own rules elsewhere (and we do, generally). But we felt strongly that we shouldn't have titles everywhere else in the book -- at a minimum, the publisher wouldn't want them in the print book format. So since we weren't including titles everywhere, we didn't feel it would be good to suggest it in Ch4.

I was originally more on the fence about this, but now I'm a solid "no titles" to avoid causing a huge transitive-closure headache elsewhere

Reply:
In the book we do have captions everywhere, which fill a similar need as the titles we are explaining to students and help clarify what is figure contains. I think this is important and we don't want student to produce charts without any explanation (whether it is a title and caption), but removing for now so that we can move forward. Follow up in https://github.com/UBC-DSCI/introduction-to-datascience-python/issues/179